pom.xml

```diff
@@ -85,20 +85,6 @@
         <maven-javadoc-plugin.version>3.3.0</maven-javadoc-plugin.version>
     </properties>
 
-    <repositories>
```

Comment:
This change is good but the pull request refers to "update some dependencies to latest bugfix versions" and this kind of change is no dependency version update.